steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt

```diff
@@ -1,7 +1,7 @@
-cd "{automationWorkerLinuxBaseDirectory}"
+cd "{automationWorkerWindowsBaseDirectory}"

```

Comment:
Replace the `{automationWorkerWindowsBaseDirectory}` with `C:\attuneautomationworker`